src/pages/groupComparison/ClinicalDataEnrichmentsTable.tsx

```diff
@@ -203,7 +203,9 @@ export default class ClinicalDataEnrichmentsTable extends React.Component<
                 paginationProps={{ itemsPerPageOptions: [20] }}
                 initialItemsPerPage={20}
                 copyDownloadProps={{
-                    showCopy: false,
+                    showCopy:
+                        false &&
```

Comment:
looks like a typo here - `false && anything === false`

Reply:
@adamabeshouse I have added this in order to remind future developers that, although inactivated here, the property is also under control of a portal.property. So, when in the future it is decided that the download control should be seen and the `false` will be turned into `true` that the statement should in fact be `!AppConfig.serverConfig.skin_hide_download_controls`.

We can debate what you prefer. There is test code that tries to spot and warn of events where the download control will show when the hide option is active.

Reply:
Ah okay, that makes sense. I think it would be clearer if you used a utility method with a comment explaining this. So

```
copyDownloadProps={{ showCopy: shouldShowCopyControls() }}
 
...

export function shouldShowCopyControls() {
    // for now, we are always hiding copy controls, but in the future we will use the app config to determine whether we
    //   show them:
    // return !AppConfig.serverConfig.skin_hide_download_controls
    return false;
}
```
what do you think?

Reply:
Yeah, that makes sense. I will implement this mechanism.